### PR TITLE
Adds Angular snippet to 'Simple component' section

### PR DESCRIPTION
**Pull request by mjrdnk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,42 @@ const module = angular.module('app.changePassword', [])
...
+import { NGXLogger } from 'ngx-logger';
+import { Auth } from 'actions/auth';
+import { Notification } from 'utils/notification';
+
+@Component({
+    selector: 'app-login',
+    templateUrl: './login.component.html',
+    styleUrls: ['./login.component.scss'],
+})
+export class LoginComponent implements OnInit {
```
Comment (feimosi) [inline README.md]: We should change the name (and paths) to `ChangePassword` 😄

File: README.md [hunk 2]
```diff
@@ -69,6 +69,42 @@ const module = angular.module('app.changePassword', [])
...
+    password: string;
+
+    constructor(
+        private logger: NGXLogger,
+        private auth: Auth,
+        private notification: Notification,
+    ) {}
+
+    ngOnInit() { 
+      this.password = '';
```
Comment (feimosi) [inline README.md]: Can this be initialised above - where its type is declared?

**Review by feimosi — COMMENTED** [review]
Awesome, thanks for the contribution!

File: README.md (hunk 2 above)
Comment (mjrdnk) [reply README.md]: Yes, this is easy, no problem. Will change tomorrow.

File: README.md (hunk 1 above)
Comment (mjrdnk) [reply README.md]: Well that's the component, not module. Should I include module declaration as well, or just rename this component?

Comment (feimosi) [reply README.md]: Hmm you can rename the component and add a simple `App` module, like in the AngularJS example above.

Comment (mjrdnk) [reply README.md]: Sure, will do.

**Comment by mjrdnk** [comment]
I have added the changes you requested.
Have a good Sunday!

**Comment by feimosi** [comment]
Thanks!

**Pull request merged**